### PR TITLE
folia

### PR DESCRIPTION
**Pull request by ewof**
it uses [UniversialScheduler](https://github.com/Anon8281/UniversalScheduler)
it's pretty hacky (my pr not the [UniversialScheduler](https://github.com/Anon8281/UniversalScheduler) library) and i have only tested on 1.20.1 u can feel free to ignore
vertical doors and sliding doors dont animate idk y, drawbridges and normal doors work just fine

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```
Comment (Anon8281) [inline .vscode/settings.json:3]: Please, do not add gitignore and IDE settings to the commit, especially in PRs.

File: .gitignore [hunk 2]
```diff
@@ -43,3 +43,5 @@ updateMirror.sh
 *dependency-reduced-pom.xml
 **/target/*
 */lib/*
+
+tmp/
```
Comment (Anon8281) [inline .gitignore:47]: Please, do not add gitignore and IDE settings to the commit, especially in PRs.

**Comment by Anon8281** [comment]
Can you try #12? Have checked it on Folia 1.19.4, but I have not used this plugin before, so there may be some problems that I missed

File: .vscode/settings.json (hunk 1 above)
Comment (ewof) [reply .vscode/settings.json:3]: ye my bad thought i removed it

**Comment by ewof** [comment]
trying #12 
it works on folia 1.20.1
the sliding doors and portcullises still snap into place instead of sliding like they do in my fork but idk how to fix, normal doors and drawbridges work fine like they do here

**Comment by ewof** [comment]
i'll close this one soon, yours is probably better for a merge because of a lower diff count
one thing though in your pr can you change setupEnv.sh to have these lines?
```
VERSIONS=(1.11.2 1.12.2 1.13 1.13.1 1.13.2 1.14.4 1.15.2 1.16.1 1.16.2 1.16.3 1.16.5 1.17.1 1.18.1 1.19 1.19.1 1.19.2 1.19.3 1.19.4 3798)
JAVA_VERSIONS=("$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_8" "$J_17" "$J_17" "$J_17" "$J_17" "$J_17" "$J_17" "$J_17" "$J_17")
```
when you run BuildTools for 1.20 it generates 1.20.1 for reason and the current script has duplicate versions and no 1.19 versions

**Comment by Anon8281** [comment]
@ewof How can I contact you? What about twitter? https://twitter.com/SevastjAnon

**Pull request closed without merging**